chef_master/source/platforms.rst

```diff
@@ -48,7 +48,7 @@ The following table lists the Foundational platforms for the chef-client and oha
      - ``5``, ``6``, ``7``
    * - Solaris
      - ``sparc``, ``x86``
-     - ``10``, ``11``
+     - ``10``, ``10 1\13`` (``10 U11``), ``11.0``, ``11.2``, ``11.3`` 
```

Comment:
Not sure how the formatting ends up with rst docs, but the end result for these should read something like:
```
10 1/13 ("U11"), 11.2, 11.3
```